lib/presentation/widgets/agenda/common_widgets.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
```

Comment:
Please split this file into multiple separated files and create a`StatelessWidget` class for each one.

By creating methods you prevents Flutter redraw optimisations.